src/JS/BigInt.js

```diff
@@ -19,6 +19,8 @@ export const fromInt = (n) => BigInt(n);
 
 export const fromTypeLevelInt = (str) => BigInt(str);
 
+export const toNumber = (n) => Number(n);
```

Comment:
I'm not quite sure about this point but I think checking is not necessary because Number() returns +infinity or -infinity (which are valid Purescript numbers) if the bounds are not checked.
By the way, no function in Data.Number (like pow or acos) does checking.
Neither does purescript-bigints' toNumber function.

Reply:
I think it's okay to return +infinity or -infinity but that should be documented. The `toNumber` export should say “If the BigInt is too positive then the result will be +Infinity. If the BigInt is too negative, then the result will be -Infinity.” or something like that.